daml/ContingentClaims/MathML.daml

```diff
@@ -54,12 +54,13 @@ instance ToXml t => ToXml (Expr t) where
     SupF t τ x -> (mo "Sup" `msub` mfenced "{" "}" bound ) :: mo "&ApplyFunction;" :: x.attribute
                   where bound = [presentation t, mo "&le;", presentation τ]
     IdentF t -> [presentation t]
-    AddF x (Cofree _ (NegF x')) -> x.attribute ++ [mo "-"] ++ x'.attribute
-    AddF x x' -> x.attribute ++ [mo "+"] ++ x'.attribute
+    SumF xs -> foldr add [] xs
```

Comment:
I should consider adding some more pattern matches here to output terms using big-sigma notation when possible. For instance, if I have a coupon paying bond, it would be nice to be able to output something like  `\sum_{i = Dec, Aug}{When i ...}`, instead of the individual terms.